src/components/DataTable/DataTable.styles.ts

```diff
@@ -30,13 +30,19 @@ export const GcxDataTableTbody = styled.tbody`
   min-width: 0;
 `;
 
-export const GcxDataTableTr = styled.tr`
+export const GcxDataTableTr = styled.tr<{onClick?: any}>`
   min-width: 0;
   border-bottom: 1px solid #ebebeb;
 
   &:last-child {
     border-bottom: none;
   }
+
+  ${(p) =>
+    p.onClick &&
+    `
+    cursor: pointer;
```

Comment:
Question; should there be some kind of hover and active styling if row is clickable?

Reply:
added hover state based on ux designs :)